aws/tf/modules/sra/variables.tf

```diff
@@ -63,161 +68,39 @@ variable "databricks_account_id" {
-  description = "Enable restrictive STS endpoint boolean flag"
-  default     = false
-}
-
 variable "enable_sat_boolean" {
-  description = "Flag for a specific SAT (Service Access Token) configuration."
-  type        = bool
-  sensitive   = true
-  default     = false
-}
-
-variable "enable_system_tables_schema_boolean" {
-  description = "Flag for enabling public preview system schema access"
+  description = "Flag to enable the security analysis tool."
   type        = bool
   sensitive   = true
   default     = false
 }
 
-variable "firewall_allow_list" {
-  description = "List of allowed firewall rules."
-  type        = list(string)
-}
-
-variable "firewall_subnets_cidr" {
-  description = "CIDR blocks for firewall subnets."
-  type        = list(string)
-}
-
-variable "hms_fqdn" {
-  type = map(string)
-  default = {
-    "ap-northeast-1" = "mddx5a4bpbpm05.cfrfsun7mryq.ap-northeast-1.rds.amazonaws.com"
-    "ap-northeast-2" = "md1915a81ruxky5.cfomhrbro6gt.ap-northeast-2.rds.amazonaws.com"
-    "ap-south-1"     = "mdjanpojt83v6j.c5jml0fhgver.ap-south-1.rds.amazonaws.com"
-    "ap-southeast-1" = "md1n4trqmokgnhr.csnrqwqko4ho.ap-southeast-1.rds.amazonaws.com"
-    "ap-southeast-2" = "mdnrak3rme5y1c.c5f38tyb1fdu.ap-southeast-2.rds.amazonaws.com"
-    "ca-central-1"   = "md1w81rjeh9i4n5.co1tih5pqdrl.ca-central-1.rds.amazonaws.com"
-    "eu-central-1"   = "mdv2llxgl8lou0.ceptxxgorjrc.eu-central-1.rds.amazonaws.com"
-    "eu-west-1"      = "md15cf9e1wmjgny.cxg30ia2wqgj.eu-west-1.rds.amazonaws.com"
-    "eu-west-2"      = "mdio2468d9025m.c6fvhwk6cqca.eu-west-2.rds.amazonaws.com"
-    "eu-west-3"      = "metastorerds-dbconsolidationmetastore-asda4em2u6eg.c2ybp3dss6ua.eu-west-3.rds.amazonaws.com"
-    "sa-east-1"      = "metastorerds-dbconsolidationmetastore-fqekf3pck8yw.cog1aduyg4im.sa-east-1.rds.amazonaws.com"
-    "us-east-1"      = "mdb7sywh50xhpr.chkweekm4xjq.us-east-1.rds.amazonaws.com"
-    "us-east-2"      = "md7wf1g369xf22.cluz8hwxjhb6.us-east-2.rds.amazonaws.com"
-    "us-west-1"      = "mdzsbtnvk0rnce.c13weuwubexq.us-west-1.rds.amazonaws.com"
-    "us-west-2"      = "mdpartyyphlhsp.caj77bnxuhme.us-west-2.rds.amazonaws.com"
-  }
-}
-
-variable "ip_addresses" {
-  description = "List of IP addresses to allow list."
-  type        = list(string)
-}
-
 variable "metastore_exists" {
   description = "If a metastore exists"
   type        = bool
 }
 
-variable "operation_mode" {
+variable "network_configuration" {
   type        = string
-  description = "The type of Operation Mode for the workspace network configuration."
+  description = "The type of network set-up for the workspace network configuration."
   nullable    = false
 
   validation {
-    condition     = contains(["sandbox", "firewall", "custom", "isolated"], var.operation_mode)
-    error_message = "Invalid operation mode. Allowed values are: sandbox, firewall, custom, isolated."
+    condition     = contains(["custom", "isolated"], var.network_configuration)
```

Comment:
to harp on this unnecessarily again, if you're just selecting from 2 options, doesn't a boolean make more sense?

Reply:
related to above greater conversation on adjusting the custom experience